mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ContainerEventHandlerAdapterBuilder.java

```diff
@@ -1,8 +1,9 @@
 package org.wildfly.extension.mod_cluster;
 
+import org.jboss.as.controller.OperationContext;
 import org.jboss.msc.service.ServiceBuilder;
 import org.jboss.msc.service.ServiceTarget;
 
 public interface ContainerEventHandlerAdapterBuilder {
-    ServiceBuilder<?> build(ServiceTarget target, String connector, int statusInterval);
+    ServiceBuilder<?> build(ServiceTarget target, OperationContext context, String connector, int statusInterval);
```

Comment:
The module that implements ContainerEventHandlerAdapterBuilder knows nothing of mod_cluster operations or its model, thus passing the OperationContext is not appropriate here.  It should instead use CapabilityServiceSupport.

This will eventually be converted to use CapabilityServiceBuilder[1].
e.g.
CapabilityServiceBuilder<?> createBuilder(String connector, int statusInterval);
[1] https://github.com/wildfly/wildfly/blob/master/clustering/common/src/main/java/org/jboss/as/clustering/controller/CapabilityServiceBuilder.java